.github/workflows/test-env-deploy.yml

```
@@ -165,12 +164,39 @@ jobs:
               for(let i=1; i<=numberOfContainers[1]; i++){
                 containers.push(i)
               }
+            echo ${{Array.from(containers.keys())}}
             return {"containers": containers}
 
       - name: echo-tags
         run: |
           echo ${{steps.get_tags.outputs.result}}
 
+  testmo-report-preparation:
+    needs: prepare-tests
+    runs-on: ubuntu-latest
```

I would recommend not automatically using the latest available major version as it could suddendly break the workflows.

I recommend setting the major version and upgrading whenever a new LTS stable is added.

```suggestion
    runs-on: ubuntu-22.04
```

---

Fixed